src/osd/ECBackend.cc

```diff
@@ -2041,21 +2078,19 @@ bool ECBackend::RMWPipeline::try_reads_to_commit()
   op->trace.event("start ec write");
 
   map<hobject_t,extent_map> written;
-  if (op->plan.t) {
-    ECTransaction::generate_transactions(
-      op->plan,
-      ec_impl,
-      get_parent()->get_info().pgid.pgid,
-      sinfo,
-      op->remote_read_result,
-      op->log_entries,
-      &written,
-      &trans,
-      &(op->temp_added),
-      &(op->temp_cleared),
-      get_parent()->get_dpp(),
-      get_osdmap()->require_osd_release);
-  }
+  op->generate_transactions(
+    op->plan,
```

Comment:
We can probably remove the members from the param list?

Reply:
Fixed.